assets/react/controllers/Delivery/DeliveryForm.jsx

```diff
@@ -472,57 +473,26 @@ export default function ({ storeId, deliveryId, order }) {
-                          <div className='mt-2 text-danger'>Editing price is not yet available in beta version.</div>
-                        </div> : null }
-                      
-                      {!deliveryId ? 
-                        <>
-                    <div className='font-weight-bold mb-2'>{deliveryId ? t("DELIVERY_FORM_NEW_PRICE") : t("DELIVERY_FORM_TOTAL_PRICE")} </div>
-                      <div>
-                        {calculatedPrice.amount
-                          ?
-                          <div>
-                            <div className='mb-1'>
-                              {money(calculatedPrice.amount)} {t("DELIVERY_FORM_TOTAL_VAT")}
-                            </div>
-                            <div>
-                              {money(calculatedPrice.amount - calculatedPrice.tax.amount)} {t("DELIVERY_FORM_TOTAL_EX_VAT")}
-                            </div>
-                          </div>
-                          :
-                          <div>
-                            <div className='mb-1'>
-                              {money(0)} {t("DELIVERY_FORM_TOTAL_VAT")}
-                            </div>
-                            <div>
-                              {money(0)} {t("DELIVERY_FORM_TOTAL_EX_VAT")}
-                            </div>
-                          </div>
-                            }
-                          <OverridePrice/>
-                      </div>
-                      {priceError.isPriceError ?
-                        <div className="alert alert-danger mt-4" role="alert">
-                          {priceError.priceErrorMessage}
-                        </div>
-                            : null}
-                        </> :
-                        null
-                        }
-
+                    <div className='order-informations__total-price border-top border-bottom pt-3 mb-4'>
+                      <ShowPrice
+                        isAdmin={isAdmin}
+                        deliveryId={deliveryId}
+                        deliveryPrice={deliveryPrice}
+                        calculatedPrice={calculatedPrice}
+                        setCalculatePrice={setCalculatePrice}
+                        priceError={priceError}
+                        setOverridePrice={setOverridePrice}
+                        overridePrice={overridePrice}
+                        priceLoading={priceLoading}
+                      />
                     </div>
 
                     <div className='order-informations__complete-order'>
                       <Button
                         type="primary"
                         style={{ height: '2.5em' }}
-                        htmlType="submit" disabled={isSubmitting || deliveryId && isAdmin === false}>
+                        htmlType="submit"
+                        disabled={isSubmitting || deliveryId && isAdmin === false}>
```

Comment:
It should also be disabled when `priceLoading` is true, as I understand https://github.com/coopcycle/coopcycle-web/issues/4854